graphql_client_codegen/src/codegen_options.rs

```diff
@@ -28,6 +30,7 @@ pub struct GraphQLClientCodegenOptions {
     /// The deprecation strategy to adopt.
     deprecation_strategy: Option<DeprecationStrategy>,
     /// Target module visibility.
+    #[derivative(Debug = "ignore")]
```

Comment:
The new crate is used to allow the "Debug" derive on this struct by ignoring this field.
Because syn's Visibility no longer derives of Debug, the code doesn't compile otherwise.

You might as well remove the Debug trait from this struct

Reply:
If we can live without the debug, let's do that. Otherwise a wrapper struct with a dummy Debug implementation would work. Sorry to bother you with that but I feel like it's worth it to keep the dependency count low.

Reply:
Alright I'll just remove the Debug trait for now.